config/application.rb

```diff
@@ -50,5 +50,17 @@ class Application < Rails::Application
 
     # Defer loading of images until it reaches a calculated distance from the viewport
     config.action_view.image_loading = "lazy"
+
+    config.view_component.default_preview_layout = "component_preview"
+    config.lookbook.project_name = "railsdevs component library"
+
+    # Tweak to Lookbook theme to render the header with the same colors as the site
+    config.lookbook.ui_theme_overrides = {
+      tw_bg_gray_800: "rgb(31 41 55 / 1)",
+      tw_text_gray_300: "rgb(209 213 219 / 1)",
+      header_bg: "var(--lookbook-tw-bg-gray-800)",
+      header_text: "var(--lookbook-tw-text-gray-300)",
+      header_border: "var(--lookbook-bg-gray-800)"
+    }
```

Comment:
I like this but maybe is not worth it to have this customization and instead we can use the zinc theme that is near to the rails devs style
![image](https://user-images.githubusercontent.com/16840674/188289598-ef1bdce3-2fb8-44f5-8e35-98b1210f8682.png)

What do you think? @joemasilotti

Reply:
Oh, don't worry about customizing the colors/theme to match RailsDevs right now. It's all changing soon anyway. Keep the defaults for now.